### DISCUSSION TITLE
Stencil Custom Extension

### DISCUSSION THREAD

**Discussion by jeremy-mgnr** [SwiftGen/SwiftGen#1130] [Q&A]
Hi, 

It is possible in StencilSwiftKit to add Stencil Extensions by calling the `stencilSwiftEnvironment` function
However SwiftGen calls this function without the possibility of adding Extensions
I am working on a custom stencil template and I would like to add filters and tags myself that I can use in my template

Are there plans to have this feature?

**Comment by teejayhh** [comment]
I am trying to figure this out as well, I also have the need for custom filters, I will report back if I find something.
